src/dataexplorer/dataexplorer-application/src/components/SchemaResponse/Schemes/Schemes.tsx

```diff
@@ -0,0 +1,281 @@
+import React, { useState, useEffect } from 'react';
+import { Collapse, Row, Popover, Button } from 'antd';
+import './Schemes.style.scss';
+import {
+    IQuestionAndAnswer,
+    IQuestion,
+    IAnswer,
+} from 'types/IQuestionAndAnswer';
+import dayjs, { Dayjs } from 'dayjs';
+
+export interface SchemesProps {
+    questions: IQuestion[];
+    answers: IAnswer[];
+    questionnaireResource: fhir.ValueSet[];
+    date: Dayjs;
+    title: string;
+}
+
+const Schemes = (props: SchemesProps) => {
+    dayjs.locale('nb');
+    const [qAndA, setQAndA] = useState<IQuestionAndAnswer[]>([]);
```

Comment:
I think unnecessary abreviation of variable names are bad practice, maybe "questionAndAnswer" etc, instead for increased readability?

Reply:
It's possible to change it, @erlinvan and I thought it would be enough since the interface tells the full name. What do you think @erlinvan ?

Reply:
I agree with @froydis98 . Since Q and A is a well known abbreviation of questions and answers, and it's type is IQuestionAndAnswer, I would argue that it is easy to understand what it is.